Content/src/Client/webpack.config.js

```diff
@@ -1,51 +1,80 @@
+    // See https://github.com/jantimon/html-webpack-plugin
+    indexHtmlTemplate: "./public/index.html",
+    fsharpEntry: "./Client.fsproj",
+    cssEntry: "./public/style.sass",
+    outputDir: "./deploy",
+    assetsDir: "./public",
+    devServerPort: 8080,
+    // When using webpack-dev-server, you may need to redirect some calls
+    // to a external API server. See https://webpack.js.org/configuration/dev-server/#devserver-proxy
+    devServerProxy: {
+        // redirect requests that start with /api/* to the server on port 8085
+        '/api/*': {
+            target: 'http://localhost:' + (process.env.SERVER_PROXY_PORT || "8085"),
+               changeOrigin: true
+           }
+       },
+    // Use babel-preset-env to generate JS compatible with most-used browsers.
+    // More info at https://babeljs.io/docs/en/next/babel-preset-env.html
+    babel: {
+        presets: [
+            ["@babel/preset-env", {
+                "targets": "> 0.25%, not dead",
+                "modules": false,
+                // This adds polyfills when needed. Requires core-js dependency.
+                // See https://babeljs.io/docs/en/babel-preset-env#usebuiltins
+                "useBuiltIns": "usage",
+            }]
+        ],
+    }
 }
 
-// passing -p on the command line to webpack specifies production mode, otherwise debug is used.
-var isProduction = process.argv.indexOf("-p") >= 0;
-
+// If we're running the webpack-dev-server, assume we're in development mode
+var isProduction = !process.argv.find(v => v.indexOf('webpack-dev-server') !== -1);
 console.log("Bundling for " + (isProduction ? "production" : "development") + "...");
 
-// Use babel-preset-env to generate JS compatible with most-used browsers.
-// More info at https://github.com/babel/babel/blob/master/packages/babel-preset-env/README.md
-var babelOptions = {
-    presets: [
-        ["@babel/preset-env", {
-            "targets": {
-                "browsers": ["last 2 versions"]
-            },
-            "modules": false,
-            "useBuiltIns": "usage",
-        }]
-    ],
-    // A plugin that enables the re-use of Babel's injected helper code to save on codesize.
-    plugins: ["@babel/plugin-transform-runtime"]
-};
+// The HtmlWebpackPlugin allows us to use a template for the index.html page
+// and automatically injects <script> or <link> tags for generated bundles.
+var commonPlugins = [
+    new HtmlWebpackPlugin({
+        filename: 'index.html',
+        template: resolve(CONFIG.indexHtmlTemplate)
+    })
+];
 
 module.exports = {
-    entry: {
-        "app": [
-            "whatwg-fetch",
-            "@babel/polyfill",
-            resolve("./Client.fsproj")
-        ]
-    },
+    // In development, bundle styles together with the code so they can also
+    // trigger hot reloads. In production, put them in a separate CSS file.
+    entry: isProduction ? {
+        app: [resolve(CONFIG.fsharpEntry), resolve(CONFIG.cssEntry)]
```

Comment:
You can simplify the entries by removing `resolve`.

Webpack expect an absolute path only for the `output.path` node.

Reply:
The `webpack.config.js` is now download from webpack-config-template repository. Actually I might consider gitignoring this file to prevent confusion

Reply:
Ah ok, so this why the `build.fsx` is modifying it to add `api` things.

Reply:
Exactly, as @alfonsogarciacaro [suggested](https://github.com/SAFE-Stack/SAFE-template/issues/194#issuecomment-444042621), we should stick to single source of truth, which I'm happy to do

Reply:
Yes i have no problem with that :)

I just review this PR as asked. We can wait to see if Alfonso wants to modify the original file with my comments or no I guess.